documents/knowledge/elements.yaml

```diff
@@ -4509,4 +4520,5 @@
   vectorise: false
   tests:
      - "[[1, 1, 1], [9, 9, 9]] : 1"
-     - "[[1, 2, '3'], [1, 2, '2']] : 0"
\ No newline at end of file
+     - "[[1, 2, '3'], [1, 2, '2']] : 0"
+
```

Comment:
Now there are two blank lines here! Most editors put one in by default, so adding one manually ends up with 2. But some editors don't, so sometimes you have to add one. Anyway, don't worry about it here, I'll put in a check and an auto-fixer

Reply:
That probably will work better than me trying to fix the newlines manually :p